ww4ff/run/train.py

```diff
@@ -60,7 +60,7 @@ def evaluate_engine(dataset: WakeWordEvaluationDataset, prefix: str, save: bool
     apb.add_options(opt('--model', type=str, choices=model_names(), default='las'),
                     opt('--workspace', type=str, default=str(Path('workspaces') / 'default')),
                     opt('--load-weights', action='store_true'),
-                    opt('--vocab', type=str, nargs='+', default=[' hey', 'fire', 'fox', 'kit', 'moxie', 'rexy', 'scout']),
+                    opt('--vocab', type=str, nargs='+', default=[' hey', 'fire fox']),
```

Comment:
is the space between fire and fox meant to be there?

Reply:
Yeah, just for backwards compatibility.